app/ui/browser-modern/setup-ipc.js

```diff
@@ -47,19 +48,25 @@ export default function({ store, userAgentClient }) {
   });
 
   ipcRenderer.on('go-back', () => {
-    if (!UISelectors.getOverviewVisible(store.getState())) {
+    const pageId = PagesSelectors.getSelectedPageId(store.getState());
+    const pageState = PagesSelectors.getPageState(store.getState(), pageId);
```

Comment:
Eh, I have no strong opinions about this. Idiomatic redux would be to not access any properties directly on the app state ever (outside of the reducers of course), and just use selectors. However, I feel like that's a bit too much hassle at times.

Our codebase is somewhat of a mixed bag, strongly but not strictly preferring selectors, so this can go either direction.